arrows/pdal/tests/test_pointcloud_io.cxx

```diff
@@ -39,78 +41,75 @@ class pointcloud_io : public ::testing::Test
 // ----------------------------------------------------------------------------
 TEST_F(pointcloud_io, create)
 {
-  using namespace kwiver::vital;
-  plugin_manager::instance().load_all_plugins();
-
-  EXPECT_NE(nullptr, algo::pointcloud_io::create("pdal"));
+  EXPECT_NE(nullptr, kv::algo::pointcloud_io::create("pdal"));
 }
 
 TEST_F(pointcloud_io, save) {
```

Comment:
I missed this style issue before.  The KWIVER style is to have each `{` and `}` on a new line.